### ISSUE TITLE
Border disappears if launch any fzf.vim command inside fzf

### ISSUE THREAD

**Issue by xinhangliu** [junegunn/fzf#1985]
- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

## Info

- OS
    - [ ] Linux
    - [x] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [ ] bash
    - [x] zsh
    - [ ] fish
    
## Problem / Steps to reproduce

1. Start `FzfCommands` (my prefix is `Fzf`), then select a command of fzf.vim
<img width="928" alt="" src="https://user-images.githubusercontent.com/21138560/79250275-fea82e00-7eb0-11ea-8fa0-d463caff67cf.png">

2. Press <kbd>C-x</kbd> to launch this command. The window border disappeared.

<img width="928" alt="" src="https://user-images.githubusercontent.com/21138560/79250518-4e86f500-7eb1-11ea-9cc6-5e8e30bd965b.png">

## My fzf config

```
" fzf {{{2
let $FZF_DEFAULT_OPTS="--layout=reverse --info=inline"
let g:fzf_layout = {
            \ 'window': {
            \ 'width': 0.64,
            \ 'height': 0.48,
            \ 'highlight': 'Identifier',
            \ 'border': 'rounded'
            \ }
            \ }
let g:fzf_history_dir = '~/.cache/fzf'
" }}}
" fzf.vim {{{2
let g:fzf_command_prefix = 'Fzf'
" }}}
```

## Debug

I have found that This issue was produced by 0f5c6e8 for fixing #1916.

**Comment by junegunn** [comment]
I can't reproduce. What's your Vim `:version`?

**Comment by xinhangliu** [comment]
> I can't reproduce. What's your Vim `:version`?

8.2.582

**Comment by xinhangliu** [comment]
Fixed by #2000.
